src/Compilers/Shared/BuildServerConnection.cs

```diff
@@ -31,12 +32,25 @@ internal sealed class BuildServerConnection
         /// </summary>
         internal static bool IsCompilerServerSupported => GetPipeName("") is object;
 
+        /// <summary>
+        /// Create a build request for processing on the server. 
+        /// </summary>
+        /// <remarks>
+        /// Even though compilation itself does not specifically require a temporary directory to be successful
+        /// this API deliberately requires <paramref name="tempDirectory"/> to have a value. The reason for this is
```

Comment:
Is this still true after the most recent commit?

Reply:
@jaredpar It looks like you didn't address this? This comment is still there.

Reply:
Had some issues where I didn't save before push and missed a few items. Cleaned most of them up. Looks like I missed this one. Will fix this in #69406.